tensorflow_datasets/translate/opus.py

```diff
@@ -0,0 +1,255 @@
+    ),
+    SubDataset(
+        name="OpenOffice",
+        description="A collection of documents from http://www.openoffice.org/.",
+        homepage="http://opus.nlpl.eu/OpenOffice-v2.php",
+        url="http://opus.nlpl.eu/download.php?f=OpenOffice/v2/moses/",
+        languages=["de", "en", "es", "fr", "jp", "sv"]
+    ),
+    SubDataset(
+        name="OpenSubtitles",
+        description="A new collection of translated movie subtitles from http://www.opensubtitles.org/",
+        homepage="http://opus.nlpl.eu/OpenSubtitles-v2018.php",
+        url="http://opus.nlpl.eu/download.php?f=OpenSubtitles/v2018/moses/",
+        languages=["af", "ar", "bg", "bn", "br", "bs", "ca", "cs", "da", "de", "el", "en", "eo", "es", "et", "eu", "fa", "fi", "fr", "gl", "he", "hi", "hr", "hu", "hy", "id", "is", "it", "ja", "ka", "kk", "ko", "lt", "lv", "mk", "ml", "ms", "nl", "no", "pl", "pt", "pt_br", "ro", "ru", "si", "sk", "sl", "sq", "sr", "sv", "ta", "te", "th", "tl", "tr", "uk", "ur", "vi", "ze_en", "ze_zh", "zh_cn", "zh_tw"]
+    )
+]}
+
+class OpusConfig(tfds.core.BuilderConfig):
+  """BuilderConfig for Opus."""
+
+  @tfds.core.disallow_positional_args
+  def __init__(self, language_pair, subsets, **kwargs):
+    """BuilderConfig for Opus.
+
+    Args:
+      language_pair: `(string, string)`, pair of languages used for translation.
```

Comment:
does ordering matter?
you should probably sort these as well

Reply:
Done.